cyclops-ui/src/components/k8s-resources/DaemonSet.tsx

```diff
@@ -44,7 +51,20 @@ const DaemonSet = ({ name, namespace }: Props) => {
     return () => {
       clearInterval(interval);
     };
-  }, [name, namespace]);
+  }, [group, kind, version, name, namespace]);
+
+  const handleRestart = () => {
+    axios
+      .post(
+        `/api/resources/restart?group=${group}&version=${version}&kind=${kind}&name=${name}&namespace=${namespace}`,
+      )
+      .then((res) => {
+        // window.location.href = "/modules/" + moduleName;
```

Comment:
Maybe we could add a [notification](https://ant.design/components/notification) that the resource was restarted successfully.

Reply:
Yep got it will work on it